cmd/micromdm/serve.go

```diff
@@ -127,15 +130,19 @@ func serve(args []string) error {
 		TLSCertPath:       *flTLSCert,
 		CommandWebhookURL: *flCommandWebhookURL,
 		NoCmdHistory:      *flNoCmdHistory,
+		UseDynSCEPChal:    *flUseDynChallenge,
```

Comment:
Nit: Chal is a odd abbreviation, would prefer the full word personally.

Reply:
(Applies to all the usages of `Chal` in this PR.

Reply:
To be perfectly honest I was annoyed at the larger diff changing the column width of the struct so made it shorter (most diff tools have 'ignore whitespace' default off, like GH. I can update it, heh.

Reply:
Yeah, it's annoying but I'd prefer that over having this variable name for eternity :p